variables.tf

```diff
@@ -53,6 +53,12 @@ variable "cluster_autoscaler_oidc_fully_qualified_subjects" {
   type        = list(string)
 }
 
+variable "cluster_service_ipv4_cidr" {
+  description = "The CIDR block to assign Kubernetes service IP addresses from."
+  default     = null
```

Comment:
Have you tested both scenarios? Passing `cluster_service_ipv4_cidr=<some valid CIDR>` and `cluster_service_ipv4_cidr=null`? How does the upstream AWS module behave in case you pass `null`?

Reply:
1. I've tested it with `cluster_service_ipv4_cidr = "172.20.0.0/16"` and it worked properly.
2. In the upstream module the default is: `cluster_service_ipv4_cidr=null` ([input_cluster_service_ipv4_cidr])(https://registry.terraform.io/modules/terraform-aws-modules/eks/aws/19.4.2#input_cluster_service_ipv4_cidr) but I haven't tested such scenario yet. Should I run such a test quickly?

Reply:
Ah ok, I think then it's absolutely fine.